src/components/CampaignTile.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import "./CampaignTile.scss";
+
+class CampaignTile extends React.Component {
+	constructor(props) {
+		super();
+	}
+
+	render() {
+		return (
+			<div className="feature-tile">
+				<div className="feature-tile-image">
+					<a href={this.props.href}>
+						<img src={this.props.imgSrc} alt="" />
+						<div className="feature-tile-content">
+							<h3>{this.props.title}</h3>
+							<p>{this.props.desc}</p>
+						</div>
+					</a>
+				</div>
+			</div>
```

Comment:
I think the ```a``` tag should go outside the ```<div className="feature-tile">```. Which will make the whole card clickable


![image](https://user-images.githubusercontent.com/20184809/42984078-100b9f5a-8c2e-11e8-8278-2c358e6cdafa.png)

Reply:
@sukhrajghuman Agree, will fix